frontend/src/Views/Login/Login.tsx

```diff
@@ -54,14 +54,16 @@ function SignIn({ classes, history }: IProps & RouteComponentProps<any>) {
   const onSubmit = (email: string, password: string) => {
     setLoading(true);
 
-    postLogin({ email, password }).then(response => {
-      if (response.status === 200) {
-        history.replace("/");
-      } else {
-        setError(true);
-      }
-      setLoading(false);
-    });
+    setTimeout(() => {
+      postLogin({ email, password }).then(response => {
+        if (response.status === 200) {
+          history.replace("/");
+        } else {
+          setError(true);
+        }
+        setLoading(false);
+      });
+    }, 1500);
```

Comment:
Geriau tipo įjunk loading animaciją tik jei load'as užtrunka ilgiau negu tarkim 300ms (i.e. lėtas connection ar kažką tokio), nes čia tai grynai švaistai laiką 😠

Reply:
As rekomenduoju pasiziureti kaip atrodo be timeout :D Labai jaggy transitionas, kai instant instat iskart